src/test-support/mocks/src/meshes/_/dataplanes/_.ts

```diff
@@ -1,7 +1,7 @@
 import type { EndpointDependencies, MockResponder } from '@/test-support'
 export default ({ fake }: EndpointDependencies): MockResponder => (req) => {
   const { mesh, name } = req.params
-  const isGateway = name.includes('gateway')
+  const isGateway = name.includes('-gateway')
```

Comment:
Thanks for adding these!

Note for future: Can we make a point of converting the other places that we do this to use `-` not `_`. I prefer `-` as it feels more URL-like , so would be good to continue to use that

Reply:
I don't think it matters too much considering this is test data, `+` is not that URL safe so we shouldn't really use it in URLs. Agree that we should just resolve this for the moment, good to chat through though